saved-sites/saved-sites-impl/src/main/java/com/duckduckgo/savedsites/impl/sync/SavedSitesSyncDataObserver.kt

```diff
@@ -52,7 +54,7 @@ class SavedSitesSyncDataObserver @Inject constructor(
 
     override fun onCreate(owner: LifecycleOwner) {
         super.onCreate(owner)
-        syncStateMonitor.syncState()
+        syncStateMonitor.syncState().distinctUntilChanged()
```

Comment:
@malmstein Can we remove the `.distinctUntilChanged()` here? or did you see something else? I assume is a leftover from previous tests, right?

Reply:
Even if not the root cause, why not having the `distinctUntilChanged()` at the source (ie. in the `syncState` implementation). 
Do we want to be emitting duplicates?

Reply:
yes, removed.

Reply:
That makes sense @aitorvs , I’ll add it to the `SyncMonitor`.